src/style-value.js

```diff
@@ -16,34 +16,55 @@
 
   function StyleValue() {}
 
-  // TODO: Add support for cssString being a sequence.
-  StyleValue.parse = function(property, cssString) {
-    if (typeof property != 'string') {
-      throw new TypeError('Property name must be a string');
-    }
-    if (typeof cssString != 'string') {
-      throw new TypeError('Must parse a string');
+  StyleValue.parse = function(property, value) {
+    if (typeof property != 'string') {    
+      throw new TypeError('Property name must be a string');   
     }
     if (!propertyDictionary().isSupportedProperty(property)) {
       // TODO: How do custom properties play into this?
       throw new TypeError('Can\'t parse an unsupported property.');
     }
 
-    // Make sure that there is no leading or trailing whitespace, case insensitive.
-    cssString = cssString.trim().toLowerCase();
-    if (propertyDictionary().isValidKeyword(property, cssString)) {
-      return new KeywordValue(cssString);
-    }
+    //Currently only supports sequences separated by ', '
+    var valueArray = value.split(', ');
+    var styleValueArray = [];
+    var supportedStyleValues =
+        propertyDictionary().getValidStyleValuesArray(property);
+
+    for (var i = 0; i < valueArray.length; i++) {
+      var cssString = valueArray[i];
+      cssString = cssString.trim().toLowerCase();
+      if (typeof cssString != 'string') {    
```

Comment:
The check that value is a string must be before you call split - otherwise you will get an exception on line 29 before you get to this line.